strings/tips/xeno.txt

```diff
@@ -64,3 +89,4 @@ As a boiler, you'll begin to glow with the more globules you have stored.
 If you see a red dot, RUN!
 Allowing marines to push caves only to block off their exit is hillariously effective.
 As a xeno, a Tier 3 unit takes up both a Tier 3 and Tier 2 slot.
+In crash, marines have landed when nuke disks appear on the map.
```

Comment:
Ehhhhh this is really a bug in my opinion and not an actual feature. I'd leave it out